docs/semgrep-assistant/getting-started.md

```diff
@@ -116,9 +113,20 @@ If [auto-triage](/semgrep-assistant/overview/#auto-triage), which allows you to
 Semgrep Assistant messages only appear in your PR comments for rules that are set to Comment or Block mode on the Rule Management page. Ensure that:
 
 * You have set rules to Comment or Block mode.
-  ![ Policies modes](/img/semgrep-assistant-comment.png)
+  ![ Policies modes](/img/semgrep-assistant-comment.png#md-width)
 * You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
 
+### Enable autofix suggestions
+
+Autofix allows you to receive code snippets to remediate true positives. Perform the following to enable it:
```

Comment:
Let's specify here that Assistant autofix and rule-based autofix are not the same?

(I made up the term rule-based, btw, because there's no term to differentiate between the two in our docs... feel free to use another term or find another differentiator.)

Reply:
WDYT about "Assistant-generated autofixes?"